src/app/fdctl/with-version.mk

```diff
@@ -1,22 +0,0 @@
-include src/app/fdctl/version.mk
-
-# Frankendancer versioning is always major 0, the first full Firedancer
-# release will be version 1.0
-FIREDANCER_VERSION_MAJOR := $(VERSION_MAJOR)
-
-# The minor version is a Firedancer specific version number, indicating
-# which release candidate branch this is. Different Firedancer release
-# branches could point to the same Solana Labs patch.
-FIREDANCER_VERSION_MINOR := $(VERSION_MINOR)$(shell printf "%02d" $(VERSION_PATCH))
-
-# For Frankendancer, we stuff the entire Solana Labs version that we are
-# linking to in the patch version.  This transforms, for example, a full
-# Solana Labs version of "1.17.8" to a minor version of "11708".
-FIREDANCER_VERSION_PATCH := $(shell grep -Po "(?<=^version = \").*(?=\")" "solana/Cargo.toml" | awk -F. '{ printf "%d%02d%02d\n", $$1, $$2, $$3 }')
-
-export FIREDANCER_VERSION_MAJOR
```

Comment:
These are used by Agave to access the version and report it to gossip etc. Can't be completely removed

Reply:
Oh nvm